backend/game/cards/keep_cards/health_manipulation_cards/test_it_has_a_child.py

```diff
@@ -0,0 +1,49 @@
+from game.cards.keep_cards.health_manipulation_cards.it_has_a_child import ItHasAChild
+from game.player.player import Player
+from game.values.locations import Locations
+
+
+def test_it_has_a_child_player_add_card(player):
+    ItHasAChild().immediate_effect(player, None)
+    assert player.has_instance_of_card(ItHasAChild())
+
+
+def test_it_has_a_child_costs_7_energy():
+    assert ItHasAChild().cost == 7
+
+
+def test_it_has_a_child_discards_all_cards_when_health_is_0(player):
+    player.current_health = 0
+    ItHasAChild().immediate_effect(player, None)
```

Comment:
I'm not a fan of card.immediate_effect meaning buy the card. It obfuscates the point of the method